app/pages/cif/project-revision/[projectRevision]/edit.tsx

```diff
@@ -83,8 +80,7 @@ export function ProjectRevisionEdit({
 }: RelayProps<{}, editProjectRevisionQuery>) {
   const router = useRouter();
   const query = usePreloadedQuery(EditProjectRevisionQuery, preloadedQuery);
-  const { session, projectRevision, allRevisionTypes, allRevisionStatuses } =
```

Comment:
allRevisionTypes seems to be being removed everywhere else, shouldn't it be removed from the query above on line 56 as well?

Reply:
Good call, It's a leftover! Thanks Josh.